### ISSUE TITLE
src/infra/readme.md doc issues ? or something else...

### ISSUE THREAD

**Issue by zebak007** [Azure/Mission-Critical-Online#1281]
I wanted to move my lab from North Europe to Sweden Central. According to src/infra/readme.md Sweden Central is supported 
but I am getting this error which looks suspicious as I would expect Azure Monitor to be available in Sweden...

Looking at https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?regions=sweden-central,sweden-south,united-kingdom-south&products=monitor  I see Prometheus is planned for Q1. Is this the root cause ?

Error: creating Account (Subscription: "8d5335cb-c495-4d8c-9ff6-d5a0ff5c3ce3"
│ Resource Group Name: "lde2efad5-monitoring-rg"
│ Account Name: "lde2efad5-swedencen-amw"): azuremonitorworkspaces.AzureMonitorWorkspacesClient#Create: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="LocationNotAvailableForResourceType" Message="The provided location 'swedencentral' is not available for resource type 'Microsoft.Monitor/accounts'. List of available regions for the resource type is 'eastus,australiasoutheast,brazilsouth,canadacentral,centralindia,centralus,eastasia,eastus2,germanywestcentral,israelcentral,italynorth,koreacentral,northeurope,norwayeast,southafricanorth,southcentralus,southeastasia,switzerlandnorth,uaenorth,uksouth,westcentralus,westeurope,westus,westus2'."
│ 
│   with module.stamp_monitoring["swedencentral"].azurerm_monitor_workspace.stamp,
│   on modules/stamp_monitoring/azure_monitor_workspace.tf line 8, in resource "azurerm_monitor_workspace" "stamp":
│    8: resource "azurerm_monitor_workspace" "stamp" {
│ 
│ creating Account (Subscription: "8d5335cb-c495-4d8c-9ff6-d5a0ff5c3ce3"
│ Resource Group Name: "lde2efad5-monitoring-rg"
│ Account Name: "lde2efad5-swedencen-amw"):
│ azuremonitorworkspaces.AzureMonitorWorkspacesClient#Create: Failure
│ responding to request: StatusCode=400 -- Original Error: autorest/azure:
│ Service returned an error. Status=400
│ Code="LocationNotAvailableForResourceType" Message="The provided location
│ 'swedencentral' is not available for resource type
│ 'Microsoft.Monitor/accounts'. List of available regions for the resource
│ type is
│ 'eastus,australiasoutheast,brazilsouth,canadacentral,centralindia,centralus,eastasia,eastus2,germanywestcentral,israelcentral,italynorth,koreacentral,northeurope,norwayeast,southafricanorth,southcentralus,southeastasia,switzerlandnorth,uaenorth,uksouth,westcentralus,westeurope,westus,westus2'."

**Comment by sebader** [comment]
Good catch! Azure Monitor Workspace was added to the reference implementation later and we didn't check the list of supported regions again. Apologies for that. So yes, you'll probably have to pick another region for the time being until Azure Monitor Workspace (managed Prometheus) is available in Sweden Central.
I'll take a note to update the doc
